db/migrate/20200513013022_update_deprecated_badge_icon_names.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class UpdateDeprecatedBadgeIconNames < ActiveRecord::Migration[6.0]
+  def up
+    migrate_value("up")
+  end
+
+  def down
+    migrate_value("down")
+  end
+
+  def migrate_value(dir)
+    icons = File.open("#{Rails.root}/lib/svg_sprite/fa4-renames.json", "r:UTF-8") { |f| JSON.parse(f.read) }
```

Comment:
I think it's OK to use a JSON file but putting it in `lib` implies it's part of the app and it's not clear it's part of a migration.

I would make a copy of the file in the migrations directory, and maybe name it the same as the migration number? `20200513013022_fa4_renames.json`

Reply:
That makes sense, thanks! I have made the change, but moved the PR back to draft for now, waiting to see whether we can do one and the same migration for group icons too (since it will use the same JSON file).